src/Ext/TechnoType/Body.h

```diff
@@ -102,6 +102,7 @@ class TechnoTypeExt
 		Valueable<bool> Ammo_Shared;
 		Valueable<int> Ammo_Shared_Group;
 
+		Valueable<bool> JumpjetFacing;
 		struct LaserTrailDataEntry
```

Comment:
```suggestion
		Valueable<bool> JumpjetFacing;
		
		struct LaserTrailDataEntry
```
Here may need a line